packages/customize-widgets/README.md

```diff
@@ -14,4 +14,16 @@ npm install @wordpress/customize-widgets
 
 _This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for ES2015+ such as IE browsers then using [core-js](https://github.com/zloirock/core-js) will add polyfills for these methods._
 
+## Technical implementation details
+
+The new Widgets Customizer replaces `Appearance > Customize > Widgets` with block editors. The original Customizer is a Backbone app, but the editor is a React app. One of the challenges is to integrate them together but make sure features from both sides still work.
```

Comment:
```suggestion
The new Widgets Customizer replaces `Appearance > Customize > Widgets` with block-based editors. The original Customizer is a Backbone app, but the new editor is a React app. One of the challenges is to integrate them together but make sure features from both sides still work.
```

Reply:
Also I think you might like to expand a bit on why we have multiple editors. It could be helpful.

Reply:
Do you mean why do we have both the Widgets screen and the Widgets Customizer? I'm not really well educated to write that though 😅, and perhaps such thing is better documented in [core article](https://wordpress.org/support/article/wordpress-widgets/) instead. This is not a new thing either, so probably developers are already familiar with this concept.